WorkloadManagementSystem/Agent/JobAgent.py

```diff
@@ -273,6 +277,12 @@ def execute( self ):
     if 'CPUTime' not in params:
       self.log.warn( 'Job has no CPU requirement defined in JDL parameters' )
 
+    # Job requirement for a number of processors
+    processors = 1
+    if 'NumberOfProcessors' in params:
+      processors = int( params['NumberOfProcessors'] )
+    wholeNode = True if 'WholeNode' in params else False
```

Comment:
And here you spell it differently, is it on purpose ? https://github.com/DIRACGrid/DIRAC/pull/3299/files#diff-6f078601851c19591c520ff380368f56R88